aten/src/ATen/native/native_functions.yaml

```diff
@@ -4560,6 +4580,7 @@
   variants: method
   dispatch:
     SparseCPU, SparseCUDA: _nnz_sparse
+    SparseCsrCPU: _nnz_sparse_csr
```

Comment:
Again, no CUDA support (and no tests for it either).

Reply:
The dispatch key has been added only for future-proofing.

Reply:
Yes, I understood that. The suggestion was made because in many cases adding SparseCsrCUDA enables functionality that contains no CUDA calls and would likely work out of the box. For instance, nnz returns just the size of values. Etc.
That said, I am fine with not exposing CUDA support when its functionality is incomplete.